mindee/documents/config.py

```diff
@@ -27,8 +27,8 @@ def check_api_keys(self) -> None:
             if not endpoint.api_key:
                 raise RuntimeError(
                     (
-                        f"Missing API key for '{endpoint.url_name} v{endpoint.version}',"
-                        "check your Client configuration.\n"
+                        f"Missing API key for '{endpoint.url_name} v{endpoint.version.strip()}',"
```

Comment:
The endpoint.version string should not need to be stripped.
If it did, none of the HTTP calls would work.